tide/core/CMakeLists.txt

```diff
@@ -93,6 +93,7 @@ list(APPEND TIDECORE_PUBLIC_HEADERS
 )
 
 list(APPEND TIDECORE_SOURCES
+  LoggingUtility.cpp
```

Comment:
the files are sorted alphabetically here, please keep that. Plus, the header file should be added to HEADERS above.